egs/swbd/s5c/local/chain/run_tdnn.sh

```diff
@@ -1 +1 @@
-tuning/run_tdnn_7e.sh
\ No newline at end of file
+tuning/run_tdnn_7g.sh
```

Comment:
I am asking this question as we will not be able to compare our results with other papers. We don't already do it anyway as we use speed-perturbation. So @tomkocse could you just add a commented line in this script

```bash
#for swbd recipe without the reverberation of training data use the following script
# it is similar to run_tdnn_7g.sh except for the run_ivector_common.sh being called.
# tuning/run_tdnn_7f.sh
```